packages/feedback-spa/src/app/modules/feedback/dropdown/dropdown.component.ts

```diff
@@ -26,7 +24,7 @@ export class DropdownComponent implements OnInit {
   @Input() truncateLength?: number;
   @Input() disableAny: boolean;
   // tslint:disable-next-line:no-output-on-prefix
```

Comment:
This line is not needed now, as you have removed `on` prefix from the selected property below this.

Reply:
Removed.